jekyll/_includes/site/header.html

```diff
@@ -1,2 +1,4 @@
 <h1>PostingPro</h1>
 <p class="tagline">Build inclusive job postings that engage quality candidates!</p>
+<p>Either start a new job posting or build from an existing template. As you type, the text will be analyzed for grade reading level and problamatic phrases that might deter the right job seeker from applying. If the reading level is too high, it will be flagged by a red line.</p>
```

Comment:
spelling: problematic

also I'm unsure about about the last sentence, it's unclear what exactly will be flagged.